ui/app/mainui/AppMain.qml

```diff
@@ -970,8 +970,9 @@ Item {
         function onAppSectionBySectionTypeChanged(sectionType, subsection, subSubsection = -1, data = {}) {
             if(!appMain.rootStore.mainModuleInst)
                 return
-
-            appMain.rootStore.mainModuleInst.setActiveSectionBySectionType(sectionType)
+            if (sectionType !== Constants.appSection.community) {
+                appMain.rootStore.mainModuleInst.setActiveSectionBySectionType(sectionType)
```

Comment:
Not the cleanest, but `setActiveSectionBySectionType` on `community` doesn't exist or do anything, it was just doing an operation for no reason. The real section change happens below in `setActiveCommunity`